.github/workflows/test.yml

```diff
@@ -0,0 +1,60 @@
+name: Python Checking
+
+on:
+  push:
+    branches:
+      - '**'
+    tags-ignore:
+      - '**'
+  pull_request:
+    branches:
+      - master
+
+
+jobs:
+  Lint:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-16.04, ubuntu-18.04, ubuntu-20.04, macos-10.15, windows-2016, windows-2019]
+        python-version: [2.7, 3.6, 3.7, 3.8, 3.9, pypy-2.7, pypy-3.6, pypy-3.7]
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+
+    - name: Linux Cache
+      uses: actions/cache@v2
+      if: startsWith(runner.os, 'Linux')
+      with:
+        path: ~/.cache/pip
+        key: ${{ runner.os }}-${{ matrix.python-version}}-${{ hashFiles('**/requirements.txt', '**/requirements-dev.txts') }}
+
+    - name: MacOS Cache
+      uses: actions/cache@v2
+      if: startsWith(runner.os, 'macOS')
+      with:
+        path: ~/Library/Caches/pip
+        key: ${{ runner.os }}-${{ matrix.python-version}}-${{ hashFiles('**/requirements.txt', '**/requirements-dev.txts') }}
+
+
+    - name: Windows Cache
+      uses: actions/cache@v2
+      if: startsWith(runner.os, 'Windows')
+      with:
+        path: ~\AppData\Local\pip\Cache
+        key: ${{ runner.os }}-${{ matrix.python-version}}-${{ hashFiles('**/requirements.txt', '**/requirements-dev.txts') }}
+
+
+    - name: Install Dependencies
+      run: |
+        pip install --upgrade pip wheel
+        pip install . -r requirements_dev.txt
+
+    - name: Tests
+      run: py.test --cov haralyzer tests/ -vv
```

Comment:
So, looks like we are running the tests via GH actions now, does this mean we could/should drop Travis CI?

Reply:
Right now travis is only used for coveralls. Working on getting coveralls to GitHub actions it will require you setting the coveralls repo token as `COVERALLS` in secrets.

Reply:
Should I do this here or should we get the repo moved first?